src/index.js

```diff
@@ -55,6 +55,8 @@ export default function (CustomElement, opts) {
 
         if (name.indexOf('on') === 0) {
           syncEvent(node, name.substring(2), props[name]);
+        } else if (name === 'style') {
+          node.setAttribute('style', props[name]);
```

Comment:
@bradleyayers Discussed IRL, but for posterity:

- React enforces you pass a style object that maps to a [CSSStyleDeclaration](https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration).
- We use `willReceiveProps()` to set properties on the element. In this lifecycle callback, `style` is passed in as an object.
- We must pass the `style` prop to `React.createElement()` as `attributes` in order for React to apply its special behaviour to it (i.e. converting to a string and numbers have `px` appended to them.